public_html/app/Controllers/Users.php

```diff
@@ -0,0 +1,141 @@
+<?php
+
+namespace VFramework\Controllers;
+
+use VFramework\Helpers\UrlHelper;
+use VFramework\Libraries\Controller;
+use VFramework\Models\User;
+use VFramework\Tools\Request;
+use VFramework\Tools\Validator;
+
+
+class Users extends Controller {
+
+    /**
+     * @var mixed
+     */
+    private $model;
+    /**
+     * @var Validator
+     */
+    public $validator;
+    /**
+     * @var Request
+     */
+    private $request;
+
+    public const LOGIN_RULES = [
+        'email' => ['required', 'exists'],
```

Comment:
```suggestion
        'email' => [
        'required',
        'exists'
        ],
```
multiple array items should look like this